FreeRTOS-Plus/Demo/coreSNTP_Windows_Simulator/SNTPClientTask.c

```diff
@@ -686,7 +687,24 @@ void calculateCurrentTime( UTCTime_t * pBaseTime,
     /* Set the current UTC time in the output parameter. */
     if( msElapsedSinceLastSync >= 1000 )
     {
-        pCurrentTime->secs = pBaseTime->secs + msElapsedSinceLastSync / 1000;
+        currentTimeSecs = ( uint64_t ) ( pBaseTime->secs ) + ( msElapsedSinceLastSync / 1000 );
+
+        /* Support case of UTC timestamp rollover on 7 February 2038. */
+        if( currentTimeSecs > UINT32_MAX )
+        {
+            /* Assert when the UTC timestamp rollover. */
+            configASSERT( currentTimeSecs > UINT32_MAX );
```

Comment:
Can we remove this assertion? Once you enter this condition, `currentTimeSecs` is larger than `UINT32_MAX`.

Reply:
The condition is wrong and updated in new commit